recipes/csaps/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "csaps" %}
+{% set version = "0.11.0" %}
```

Comment:
Latest version is 1.1.0, right? Any reason why you are adding an old one?

Reply:
I am resurrecting this recipe from @kalling , which has worked in the past for us.  I hoped we could merge this version, then let the autotick-bot file a PR for a new version on the new feedstock.